src/test/ml-config/security/users/test-geo-data-services-reader.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
This and the other deleted user weren't being used for anything, so deleted them to minimize confusion.